playbooks/roles/ecomworker/defaults/main.yml

```
@@ -26,6 +26,7 @@ ECOMMERCE_WORKER_REPOS:
 # Requires that New Relic be enabled via COMMON_ENABLE_NEWRELIC, and that
 # a key be provided via NEWRELIC_LICENSE_KEY.
 ECOMMERCE_WORKER_NEWRELIC_APPNAME: '{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-{{ ecommerce_worker_service_name }}'
+ECOMMERCE_WORKER_ENABLE_NEWRELIC_APP: true
```

May want to default this to the common value, just on principle, although I don't think it will matter.

```suggestion
ECOMMERCE_WORKER_ENABLE_NEWRELIC_APP: '{{ COMMON_ENABLE_NEWRELIC_APP }}'
```